src/test/resources/application.properties

```diff
@@ -1,6 +1,6 @@
 # Change port to avoid messing with local Kafka instance
-quarkus.http.port=8085
-quarkus.http.test-port=8085
+quarkus.http.port=9085
```

Comment:
Why is that?

Reply:
To allow tests to run while running the application. If not, the tests server will attempt to use the same port.

Reply:
But you are changing both test-port and normal port.
Btw. In policies-backend I don't set the test-port and the Quarkus-Test just does the right thing. Something to consider.

Reply:
> But you are changing both test-port and normal port.

This is on `test/application.properties`

> Btw. In policies-backend I don't set the test-port and the Quarkus-Test just does the right thing. Something to consider.

I suppose is related to the testing setup. In policies it works for me :)

Reply:
Ah right, test/application.properties